channels/rdpgfx/server/rdpgfx_main.c

```diff
@@ -1213,6 +1272,8 @@ static UINT rdpgfx_recv_cache_import_offer_pdu(RdpgfxServerContext* context, wSt
  */
 static UINT rdpgfx_recv_caps_advertise_pdu(RdpgfxServerContext* context, wStream* s)
 {
+	if (checkCapsAreExchanged(context))
+		return ERROR_ALREADY_INITIALIZED;
```

Comment:
This check here (in `rdpgfx_recv_caps_advertise_pdu`) needs to be removed, because this is actually allowed here.